pkg/validation/validation.go

```diff
@@ -4,6 +4,26 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"regexp"
+	"strconv"
+
+	"k8s.io/apimachinery/pkg/version"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+var (
+	loggerValidation = ctrl.Log.WithName("validation")
+	//IsOutOfServiceTaintSupported will be set to true in case OutOfServiceTaint is supported (k8s 1.26 or higher)
+	IsOutOfServiceTaintSupported bool
```

Comment:
Changed to pass OutOfServiceTaintSupported info instead of global var based on https://github.com/medik8s/node-maintenance-operator/pull/110/commits/472dae39e5f054ece3448b7c4f1989e6d95bb86e.
[This](https://github.com/medik8s/fence-agents-remediation/pull/92/commits/f54978f9be62979cc0e40bbdf34a482208ccb831) is the changes to do that.